components/eam/tools/topo_tool/orographic_drag_toolkit/cube_to_target.F90

```diff
@@ -0,0 +1,2550 @@
+!
```

Comment:
Why are we adding a second version of cube_to_target? 
This seems like a recipe for diverging versions where one is working and the other is broken, right?
I feel like we should only ever have a single version.

Reply:
This is a suite of toolkit for generation of additional topographic parameters for the new scheme (documented [here](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/4625563753/Generation+of+the+additional+topographic+parameters+for+the+new+orographic+drag+scheme+considering+orographic+anisotropy+for+all+flow+directions)).
The derivation code is developed before also based on cube_to_target. We'd want to make an independent workflow so as to be separate from the workflow now maintained by Mark's @mt5555.
As documented in the confluence ([here](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/4625563753/Generation+of+the+additional+topographic+parameters+for+the+new+orographic+drag+scheme+considering+orographic+anisotropy+for+all+flow+directions)), we'd generate the additional topographic parameters and append them to the topo data that was generated by the default topo workflow in E3SM.

Reply:
you should name the tool something else then - having two tools with the same name is definitely going to lead to confusion.

But also - if these quantities need to be appending to a topo file, why not enhance the existing cube_to_target tool to produce the quantities and not make more steps for people to work through when they create new grids? This seems like it's going to make everything harder to maintain rather than working within the existing workflows.